workflows/energyplus.py

```diff
@@ -230,7 +233,7 @@ def run_energyplus(self, isIP, run_directory, file_name, args):
                         if os.path.exists(ipmtr_path):
                             shutil.copy(ipmtr_path, mtr_path)
                             os.replace(ipmtr_path, eplusoutmtr_path)
-                    os.remove(converttxt_run_path)
+                        os.remove(converttxt_run_path)
```

Comment:
If you look above, `converttxt_run_path` was only declared inside the `if os.path.exists(convertESOMTR_binary)` block, and this delete call was outside of that block, so this was quietly failing.